src/main/webapp/js/instructorHome.es6

```diff
@@ -107,3 +114,57 @@ function instructorHomeDateComparator(x, y) {
     }
     return x0 < y0 ? -1 : 0;
 }
+
+function bindCoursePanels() {
+    const panels = $('div.panel');
+    let numPanels = 0;
+    for (let i = 0; i < panels.length; i += 1) {
+        const heading = $(panels[i]).children('.panel-heading');
+        const bodyCollapse = $(panels[i]).children('.panel-collapse');
+        if (heading.length !== 0 && bodyCollapse.length !== 0) {
+            $(heading[0]).attr('data-target', `#panelBodyCollapse-${numPanels}`);
```

Comment:
Maybe use ``.data('target', `#panelBodyCollapse-${numPanels}`);`` instead. This could be used for other `data` attributes too.